pkg/build/strategies/layered/layered.go

```diff
@@ -63,14 +64,24 @@ func (b *Layered) CreateDockerfile(config *api.Config) error {
 	}
 
 	buffer.WriteString(fmt.Sprintf("FROM %s\n", b.config.BuilderImage))
-	buffer.WriteString(fmt.Sprintf("COPY scripts %s\n", locations[0]))
+	// if -s / --scripts / --scripts-url are not specified, there will not be a scripts subdir
+	// off of the working dir, so don't copy here, and we'll have to tweak the scripts URL to use the env var
+	// STI_SCRIPTS_PATH
+	scriptsIncluded := len(config.ScriptsURL) > 0
```

Comment:
there are two ways to inject scripts other than via the image itself.
1) scriptsurl (covered here)
2) scripts included in the source repo itself. seems like this check would break that flow?